commands/init_test.go

```diff
@@ -1,32 +1,92 @@
 package commands
 
 import (
-	"github.com/github/hub/Godeps/_workspace/src/github.com/bmizerany/assert"
-	"github.com/github/hub/github"
+	"path/filepath"
+	"fmt"
 	"os"
-	"regexp"
 	"testing"
+
+	"github.com/github/hub/Godeps/_workspace/src/github.com/bmizerany/assert"
+	"github.com/github/hub/github"
 )
 
-func TestTransformInitArgs(t *testing.T) {
+func setup() {
```

Comment:
It doesn't seem that Go `testing` package exposes any built-in way of declaring setup/teardown blocks for the whole suite.